src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/types.js

```diff
@@ -121,12 +121,17 @@ export interface FormStoreInterface {
     +options: SchemaOptions,
     +resourceKey: ?string,
     +schema: Object,
+    +set: (name: string, value: mixed) => void,
     +setMultiple: (data: Object) => void,
     +setType: (type: string) => void,
     +types: {[key: string]: SchemaType},
     +validate: () => boolean,
 }
 
+export type FieldChangeContext = {
+    isDefaultValue?: boolean,
```

Comment:
maybe call this `markFormAsDirty`

Reply:
I would go the other way `markFormNotDirty: true` as markFormAsDirty should be the default.

Reply:
I'm also okay to keep it this way, so we are maybe flexible how to behave on default vs server values.